presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/thrift/MockHiveMetastoreClient.java

```diff
@@ -167,8 +172,8 @@ public Table getTable(String dbName, String tableName)
         }
 
         return new Table(
-                TEST_TABLE,
-                TEST_DATABASE,
+                tableName,
+                dbName,
```

Comment:
I think we don't need to change TEST_DATABASE to dbName, since we are not allowing any other dbName via the if condition above.